### PR TITLE
Sole survivor gets hijack items

### LINKED ISSUE #17442: Sole Survivor objective restricts Hijack uplink items
### Severity

Inconvenience

### Short Description

If you get the objective to escape on the shuttle with nobody else, you can't get uplink items that require hijack despite sole survivor being basically the same thing. (and then you gotta bug the admins to TC trade it for you)

### Reproduction

1. be traitor with sole survivor objective
2. attempt to buy hijack-only item (ex: romerol)
3. your objectives do not permit this item

### Round ID

44835

### Test Merges

_No response_

### BYOND Username

Jachlompsky

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by ReddicusDragon**
# Document the changes in your pull request
i hate council

closes #17442 
# Wiki Documentation
Romerol, roburger, singulo beacon and sm delaminator are given to sole survivor
# Changelog
:cl:  
tweak: sole survivor now has access to hijack items  
/:cl:

### CODE REVIEW

**Comment by ynot01** [comment]
sole survivor ought to be a subtype of hijack methinks

**Comment by ReddicusDragon** [comment]
> sole survivor ought to be a subtype of hijack methinks

I just fix issues man 😔

**Comment by JamieD1** [comment]
The fuck is sole survivor

**Comment by ReddicusDragon** [comment]
Jamie moment

**Review by Moltijoe — CHANGES_REQUESTED** [review]
should probably make it just pretend hijack also counts as sole survivor
I can't think of any reason we'd want an item that can go to one but not the other

File: yogstation/code/game/gamemodes/objective.dm [hunk 1]
```diff
@@ -5,15 +5,13 @@ GLOBAL_LIST_INIT(infiltrator_objective_areas, typecacheof(list(/area/yogs/infilt
...
-/datum/objective/sole_survivor
+/datum/objective/hijack/sole_survivor
 	name = "sole survivor"
 	explanation_text = "Escape on the shuttle to ensure <b>no one except you</b> escapes alive and out of custody."
 	team_explanation_text = "Escape on the shuttle to ensure <b>no one except your team</b> escapes alive and out of custody. Leave no team member behind."
 	martyr_compatible = 0 //Technically you won't get both anyway.
 
-/datum/objective/sole_survivor/check_completion() // Requires all owners to escape.
-	if(..())
-		return TRUE
```
Comment (Moltijoe) [inline yogstation/code/game/gamemodes/objective.dm:16]: Have it be
`if(completed) return TRUE`
to allow for admin override

**Review by Moltijoe — CHANGES_REQUESTED** [review]

**Review by Moltijoe — APPROVED** [review]
I don't like your tone young man

**Pull request merged**